### PR TITLE
docs: meeting minutes 2023-04-11

### PR DESCRIPTION
**Pull request by UlisesGascon**
close https://github.com/nodejs/build/issues/3297

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2023-04-11.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  * We will remove this item from the agenda
+  * We are in the newest version for most of the machines already
+  * Richard will close the issue soon
+* Support for llnode builds [#3017](https://github.com/nodejs/build/issues/3017)
+  * We need additional information to see if we can provide support
+* Onboarding F3n67u to the build wg [#3003](https://github.com/nodejs/build/issues/3003)
+  * It was a misunderstanding on the onboarding process
+  * Ulises will close the issue
+* Interest in establishing a "Build Helper" role [#2550](https://github.com/nodejs/build/issues/2550)
+ * There is an ansible tower/AWX instance that can be used to run ansible playbooks against the jenkins agents.
```
Comment (nschonni) [inline doc/meetings/build-wg-meeting-2023-04-11.md:79]: These bullets didn't get indented enough, so the rendering is off

**Review by mhdawson — APPROVED** [review]
LGTM

File: doc/meetings/build-wg-meeting-2023-04-11.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  * We need to change how we access to Cloudflare from a single account to a individual accounts (so it will be easier to revoke access)
+  * We need to explore how to flexible is the Cloudflare system in order to grant read/write access to specific individuals to avoid accidental changes or modifications as it is hard to rollback changes
+  * The access to Ovflowd won’t be granted yet, as we will first need to provide support for individual accounts.
+  * Richard potentially can lead the initiative
+* Membership audit [#3144](https://github.com/nodejs/build/issues/3144)
+  * We are not ready to make any changes
+  * We will keep the topic in the Agenda
+* Disk full on downloads server [#3125](https://github.com/nodejs/build/issues/3125)
+  * The server was full due nightly builds
+  * Richard made an script to manual clean the builds (PR ongoing)
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2023-04-11.md:52]:
```suggestion
  * Stewart made an script to manual clean the builds (PR ongoing)
```

File: doc/meetings/build-wg-meeting-2023-04-11.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  * The server currently is half full (not a risk currently)
+  * Ulises will remove this topic from the agenda
+* Retiring the Raspberry Pi cluster [#3102](https://github.com/nodejs/build/issues/3102)
+  * Only the Inventory clean up is pending
+* Improve Jenkins alerts and reporting [#3088](https://github.com/nodejs/build/issues/3088)
+  * This issue was related to a POC that monitors the inventory machine by doing a ping and expose the data in Grafana
+  * Ulises changed the approach to follow something similar as what has been implemented in  the Security WG for the OpenSFF Scorecard Monitoring with Github actions but using directly the Jenkins API.
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2023-04-11.md:59]:
```suggestion
  * Ulises changed the approach to follow something similar as what has been implemented in the Security WG for the OpenSSF Scorecard Monitoring with GitHub actions but using directly the Jenkins API.
```

**Review by targos — APPROVED** [review]

**Pull request merged**